test/e2e/pod_clone_test.go

```diff
@@ -155,4 +156,39 @@ var _ = Describe("Podman pod clone", func() {
 		Expect(strings[0]).Should(ContainSubstring("size=10240k"))
 	})
 
+	It("podman pod create --uts test", func() {
+		SkipIfRemote("hostname for the custom NS test is not as expected on the remote client")
```

Comment:
I don't know what that means. The new flag does not seem to be tested on the remote client which is not ideal.

Reply:
on the remote client tests, a hostname like `fedora` does not match `os.Hostname` from the tests. I guess I can modify the test to still run remotely but skip that check.

Reply:
ok, sgtm! i will push a final version of this in a few minutes